pkgs/games/openxray/default.nix

```diff
@@ -49,20 +52,30 @@ stdenv.mkDerivation rec {
   cmakeBuildType = "RelWithDebInfo";
   dontStrip = true;
 
+  makeWrapperArgs = lib.optionals stdenv.hostPlatform.isLinux [
+    # Needed because of dlopen module loading code
+    "--prefix LD_LIBRARY_PATH : $out/lib"
+  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
+    # Because we work around https://github.com/OpenXRay/xray-16/issues/1224 by using GCC,
+    # we need a followup workaround for Darwin locale stuff when using GCC:
+    # runtime error: locale::facet::_S_create_c_locale name not valid
+    "--run 'export LC_ALL=C'"
+  ];
+
   postInstall = ''
-    # needed because of SDL_LoadObject library loading code
-    wrapProgram $out/bin/xr_3da \
-      --prefix LD_LIBRARY_PATH : $out/lib
+    wrapProgram $out/bin/xr_3da ${toString finalAttrs.makeWrapperArgs}
```

Comment:
This should probably be a bash array instead

Reply:
Will prolly just reduce this to Darwin-only wrapper to set `LC_ALL` for the locale issue. For my own unterstanding though, could you explain what you mean by that? Manage `makeWrapperArgs` fully in bash as an array instead of Nix?